MonkeyWrench.Web.WebService/WebServices.asmx.cs

```diff
@@ -813,6 +813,63 @@ FROM RevisionWork
 		}
 
 		[WebMethod]
+		public GetAuditResponse GetAuditHistory (WebServiceLogin login, int limit, int offset)
+		{
+			GetAuditResponse response = new GetAuditResponse ();
+
+			using (DB db = new DB ()) {
+				Authenticate (db, login, response, true);
```

Comment:
I think this should be limited to administrators, so instead use:

    VerifyUserInRole (db, login, Roles.Administrator);